japitana_johnwalter/style.css

```diff
@@ -0,0 +1,314 @@
+@font-face {
+  font-family: "ProggySquareTT";
+  src: url("https://db.onlinewebfonts.com/t/5423e3f6b569eee8bb7179f1319240ea.eot");
+  src: url("https://db.onlinewebfonts.com/t/5423e3f6b569eee8bb7179f1319240ea.eot?#iefix")
+      format("embedded-opentype"),
+    url("https://db.onlinewebfonts.com/t/5423e3f6b569eee8bb7179f1319240ea.woff2")
+      format("woff2"),
+    url("https://db.onlinewebfonts.com/t/5423e3f6b569eee8bb7179f1319240ea.woff")
+      format("woff"),
+    url("https://db.onlinewebfonts.com/t/5423e3f6b569eee8bb7179f1319240ea.ttf")
+      format("truetype"),
+    url("https://db.onlinewebfonts.com/t/5423e3f6b569eee8bb7179f1319240ea.svg#ProggySquareTT")
+      format("svg");
+}
+
+@media (min-width: 426px) {
+  .container {
+    display: grid;
+    grid-template-areas:
+      "itm1 itm2"
+      "itm3 itm4"
+      "itm5 itm5"
+      "itm6 itm6"
+      "itm7 itm7";
+  }
+
+  iframe {
+    width: 50%;
+  }
+}
+
+@media (max-width: 426px) {
+  .container {
+    display: flex;
+    flex-direction: column;
+  }
+
+  iframe {
+    width: 90%;
+  }
+}
+
+* {
+  font-family: "ProggySquareTT";
+  font-weight: 400;
+  font-style: normal;
+  color: #eeeeee;
+}
+
+body {
+  font-family: Arial, sans-serif;
+  margin: 20px;
+  background-color: #222831;
+}
+
+img {
+  width: 200px;
+  height: 160px;
+  opacity: 0.5;
+  object-fit: contain;
+  max-width: 100%;
+  border-radius: 8px;
+}
+
+img:hover {
+  opacity: 1;
+}
+
+form {
+  border: 1px solid #00adb5;
+  width: 75%;
+}
+
+.ul-class {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #393e46;
+}
+.flag-img {
```

Comment:
Separate every css block with a single space.
Add a single space after the selector/s and before "{".

Follow the coding guidelines:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwAR2l29lcF09B6N3jV9JjsQn2kIge3yJWLvkiVWepwKW1vg5YT2a6o6bI2Pw#slide=id.g24acb740da7_0_35